ext/tmpfiles/pe-bolt-server.conf

```diff
@@ -1 +0,0 @@
-d /var/run/puppetlabs/bolt-server 0755 pe-bolt-server pe-bolt-server -
```

Comment:
Why don't we need this anymore?

Reply:
On systemd, I switched back to running the server foreground (simple mode) rather than having it fork. So no more need for a pidfile.

Reply:
https://github.com/puppetlabs/bolt-vanagon/pull/45 explained